tmpl/x_helpers_test.html

```diff
@@ -939,7 +939,7 @@
     t.Errorf("expected invalid, got %v", bb.Valid)
```

Comment:
I think it'd be worth adding all of the various tests for our new bool type mate (incl. explicit `null` etc)